extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/index.tsx

```diff
@@ -3,9 +3,17 @@ import PropTypes from 'prop-types';
 //
 import PanelStudyBrowserTracking from './PanelStudyBrowserTracking';
 import getImageSrcFromImageId from './getImageSrcFromImageId';
-import getStudiesForPatientByStudyInstanceUID from './getStudiesForPatientByStudyInstanceUID';
 import requestDisplaySetCreationForStudy from './requestDisplaySetCreationForStudy';
 
+function _getStudyForPatientUtility(extensionManager) {
+  const utilityModule = extensionManager.getModuleEntry(
```

Comment:
The function from the utility module is so small that it feels weird to me to have this dependency. No change needed for now, but I'm starting to be wary of all of the inter-extension dependencies. You now have to use the default extension in your app for this tiny utility. But I get the measurement-tracking extension is highly coupled with the mode, and the mode uses a default so... eh. But it still feels bad and we shouldn't try to make this commonplace.